tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/ViaInteractive.fsx

```diff
@@ -0,0 +1,30 @@
+open System.Runtime.CompilerServices
+open CSharpLib
+;;
+type MyTy() =
+    static member GetCallerFilePath([<CallerFilePath>] ?path : string) =
+        path
+    static member GetCallerLineNumber([<CallerLineNumber>] ?line : int) =
+        line
+;;
+let matchesPath path (s : string) =
+    s.EndsWith(path)
+     && not (s.Contains("\\\\"))
+     && not (path.Contains("\\.\\"))
+     && not (path.Contains("\\..\\"))
+
+let scriptName = if Array.contains "--exec" (System.Environment.GetCommandLineArgs()) then "ViaInteractive.fsx" else "stdin"
+let checkPath = sprintf "Conformance\\SpecialAttributesAndTypes\\Imported\\CallerInfo\\%s" scriptName
+;;
```

Comment:
No need for `;;` I think.  Or use `let _ = ...` or some other formulation if needed?